<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SchedulerClass</class>
 <widget class="QDialog" name="SchedulerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>644</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCalendarWidget" name="calendarWidget">
     <property name="firstDayOfWeek">
      <enum>Qt::Sunday</enum>
     </property>
     <property name="horizontalHeaderFormat">
      <enum>QCalendarWidget::ShortDayNames</enum>
     </property>
     <property name="verticalHeaderFormat">
      <enum>QCalendarWidget::NoVerticalHeader</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTimeEdit" name="timeEdit">
     <property name="displayFormat">
      <string>h:mm:ss AP</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="scheduler_checkBox">
     <property name="text">
      <string>Scheduler Enable</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="radio_groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QRadioButton" name="weekly_radioButton">
        <property name="text">
         <string>Weekly</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="oneTime_radioButton">
        <property name="text">
         <string>One Time</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="weekDay_groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="sunday_checkBox">
        <property name="text">
         <string>Sunday</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="monday_checkBox">
        <property name="text">
         <string>Monday</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="tuesday_checkBox">
        <property name="text">
         <string>Tuesday</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="wednesday_checkBox">
        <property name="text">
         <string>Wednesday</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="thursday_checkBox">
        <property name="text">
         <string>Thursday</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="friday_checkBox">
        <property name="text">
         <string>Friday</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="saturday_checkBox">
        <property name="text">
         <string>Saturday</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
